src/engraving/dom/volta.cpp

```diff
@@ -360,7 +360,11 @@ PointF Volta::linePos(Grip grip, System** system) const
         x += (isAtSystemStart ? 0.5 : -0.5) * absoluteFromSpatium(lineWidth());
     } else {
         if ((*system) && segment->tick() == (*system)->endTick()) {
-            x += segment->staffShape(backSegment()->effectiveStaffIdx()).right();
+            staff_idx_t si = backSegment()->effectiveStaffIdx();
+            if (si == muse::nidx) {
+                return PointF(x, 0.0);
```

Comment:
My mistake, thanks. `muse::nidx` is a valid return value from `effectiveStaffIdx` so it should always be checked (in which case it's irrelevant what we return here because this element will not be drawn anyway.